lib/simp/cli/commands/command_family.rb

```diff
@@ -11,10 +23,13 @@ def sub_commands
     return @sub_commands if @sub_commands
     @sub_commands = {}
     subcmd_files = Dir.glob(File.expand_path("#{snakecase_name}/*.rb", __dir__)).sort_by(&:to_s)
-    subcmd_files.each { |file| require file }
-    Simp::Cli::Commands::Environment.constants.each do |constant|
+
+    self.class.constants.each do |constant|
+      obj = self.class.const_get(constant)
```

Comment:
why bother with this var? Can just put that line in place of what you took out below.

Reply:
Line 29 needs `obj` as well.